File: IT/Stringhe/Java/Substring.java
/* file Substring.java */

public class Substring {
	public static void main(String [] args) {
		String s = "Ciao, mondo!";

		String s1 = s.substring(0, 6);
		String s2 = s.substring(6, s.length());

		System.out.println(s1);
		System.out.println(s2);
	}
}

File: IT/Stringhe/C++/substring.cpp
/* file substring.cpp */

#include <iostream>

using std::cout;
using std::endl;
using std::string;

int main()
{
	string s = "Ciao, mondo!";

	string s1 = s.substr(0, 6);
	string s2 = s.substr(6, s.size());

	cout << s1 << endl;
	cout << s2 << endl;
}

